crates/ruff/src/rules/flake8_pyi/rules/unnecessary_literal_union.rs

```diff
@@ -0,0 +1,92 @@
+use ruff_python_semantic::SemanticModel;
+use rustpython_parser::ast::{self, Expr, Operator, Ranged};
+
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks for the presence of multiple literal types in a union.
+///
+/// ## Why is this bad?
+/// Literal types accept multiple arguments and it is clearer to specify them as a single literal.
```

Comment:
Annoyingly, we try to wrap these at 80 characters so that they stick to that maximum width when rendered in the terminal :(